.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,21 @@
+name: Run benchmark
+
+on: push
+
+jobs:
+  benchmark:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-latest, ubuntu-latest]
```

Comment:
@jgonggrijp If you want to use the GitHub actions setup to run benchmarks against more combos of node versions and OSes already, feel free to extend this.

Reply:
Nah, I kind of trust that Node.js code is cross-platform. Also, I'm not interested in using results from a CI container, so I rather save some CPU cycles.

Reply:
Good idea to test compatibility in this way, though!

Reply:
I'd argue that a CI container is the real world as well, so it might be interesting to know about the results too, even if they are not the most important ones. I'll leave it up to.you.